src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
<img width="865" alt="Screenshot 2022-12-14 at 16 00 20" src="https://user-images.githubusercontent.com/94289038/207614720-41c4b623-8c3d-47fc-ab35-33b611d20c63.png">

where are modifiers?
also they should work